### ISSUE TITLE
setup.py

### ISSUE THREAD

**Issue by robraddi** [vvoelz/nmr-biceps#35]
[Here is a nice tutorial that explains how to upload your setup.py to PyPI](https://marthall.github.io/blog/how-to-package-a-python-app/)

**Comment by yunhuige** [comment]
Awesome! It's really much easier than what I thought. Is "conda install" similar to this? @robraddi

**Comment by robraddi** [comment]
@yunhuige Yes, it actually uses PyPI. [Here is the link](https://conda.io/docs/user-guide/tutorials/build-pkgs-skeleton.html)

**Comment by yunhuige** [comment]
@robraddi Great! As long as we get the scripts ready, it will be very straightforward to get the whole thing done!
